### PR TITLE
Darktheme

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
Updated dark theme with contexts
image
image
image

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -41,6 +41,17 @@ body {
...
+  --primary-dark-color: 196, 79%, 56%;
+  --complementary-light-color: 16, 79%, 76%;
+  --complementary-color: 16, 79%, 66%;
+  --complementary-dark-color: 16, 79%, 56%;
+  --red-800: 0, 74%, 54%;
+  --grey-500: 210, 11%, 71%;
+  --green-500: 125, 100%, 39%;
+  --darktheme-primary-color: 220, 15%, 16%;
+  --darktheme-secondary-color: 217, 19%, 27%;
+
```
Comment (Naveenchandar) [inline src/App.css]: Please use hex codes for consistency.

File: src/App.js [hunk 2]
```diff
@@ -29,7 +31,8 @@ function App() {
   };
 
   return (
-    <div className="App">
+    <div className={`${theme ? "background__dark" : "background__light"}`}>
+      {" "}
```
Comment (Naveenchandar) [inline src/App.js]: You can remove this.

File: src/components/reducers/quizReducer.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
 export const quizReducer=(quizState,action)=>{
     console.log(action.payload);
```
Comment (Naveenchandar) [inline src/components/reducers/quizReducer.js]: Remove console.log

File: src/components/Category/Category.css [hunk 4]
```diff
@@ -12,9 +9,31 @@
...
+.setting__cat{
+    background-color: var(--bg);
+  color: var(--black-bg);
+  height: 100vh;
+}
 
 .btns-explore{
     width: 60%;
     border-radius: 12px !important; 
     margin: 1rem auto !important;
```
Comment (Naveenchandar) [inline src/components/Category/Category.css:21]: Please avoid using important.

**Review by Naveenchandar — APPROVED** [review]
Good breakdown of components.
Nice use of theme context.
Folder structure looks good.

File: src/components/Questions/Questions.js [hunk 5]
```diff
@@ -46,9 +46,10 @@ export const Questions = ({
   };
 
   return (
-    <div className="question">
+    <div className="question_section">
+      <div className="question">
       <div className="singleQuestion">
-        <div className="questionss">{questions[currQues].question}</div>
+        {/* <div className="questionss">{questions[currQues].question}</div> */}
```
Comment (sakshi006) [inline src/components/Questions/Questions.js]: You can remove the comments

**Review by sakshi006 — APPROVED** [review]
1. I think, While entering the name and difficulty, the text alongside the input element must be placed a little bit upwards. 
2. The transitions are smooth and nice.
3.  The UI is nice and engaging.
4.  I liked your code for theme context, it's clean and understandable. Got to learn somehting new.

**Review by shilpe26 — APPROVED** [review]
▪️Good looks prettier, the good naming convention used
▪️See some previous suggestions already given then you are good to merge

**Pull request merged**